ui/src/utils/ajax.js

```diff
@@ -72,9 +72,8 @@ const AJAX = async (
     // AJAX request and action creator.
     return links ? generateResponseWithLinks(response, links) : response
   } catch (error) {
-    const {response} = error
-
-    throw links ? generateResponseWithLinks(response, links) : response // eslint-disable-line no-throw-literal
```

Comment:
removing this breaks the application when run with OAuth configured -- the login screen won't link without `links` on the error object. does removing this pertain to the story being tackled in this PR?

i agree with the intention of removing `links` from here, but this seems like something to tackle in a separate issue and PR and we can think through how to account for what would otherwise break by its removal.